### DISCUSSION TITLE
Quirks mode

### DISCUSSION THREAD

**Discussion by Nikolas-Dragolas** [SafeExamBrowser/seb-win-refactoring#301] [Q&A]
Hello, 

First of all thanks for this fine software and all developers.

We are updated our SEB from 2.4.1 to 3.1.1 (about hundred Win-10 PCs). When we tested all new versions 3.x.x we get a problem with one of our LAN repositories(pictures).
A messages occurred:  _No suitable ID found!_. The developer console show one issue: _Page layout may be unexpected due to Quirks Mode_
With the old version 2.4.1 the repository work like a charm.
Is that a problem with our server or with SEB? Can you guide us on how to solve this problem?

Regards, 

Nikolas

**Comment by danschlet** [comment]
Could it be that your web application is slightly outdated for modern (nowadays) browsers?

https://developer.mozilla.org/en-US/docs/Web/HTML/Quirks_Mode_and_Standards_Mode

"In quirks mode, layout emulates nonstandard behavior in Navigator 4 and Internet Explorer 5."

**Reply by Nikolas-Dragolas** [reply]
Thank you for quick reply  danschlet!

**Reply by dbuechel** [reply]
You might find further information in the browser log of the affected session, to be found under `%LocalAppData%\SafeExamBrowser\Logs\..._Browser.log`.
